### PR TITLE
Another way to call parent initialise in viewmodels

### PR DESCRIPTION
**Pull request by anstaendig**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,13 +1,13 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.1.2-5'
+    ext.kotlin_version = '1.1.3'
     repositories {
         jcenter()
-        maven { url 'https://maven.google.com' }
+        google()
```
Comment (jonduran) [inline build.gradle]: looks better! is this new?

**Comment by jonduran** [comment]
👍

**Comment by jonduran** [comment]
oh, wait... do we need 👍  :D :D :D

File: build.gradle (hunk 1 above)
Comment (anstaendig) [inline build.gradle]: I don't think it is. I think I recognised it yesterday in a talk I watched from I/O

**Comment by anstaendig** [comment]
We don't need the thumbs, but I'll take them.

**Pull request merged**